apps/google-analytics-4/lambda/src/middlewares/verifySignedRequests.ts

```diff
@@ -38,7 +38,7 @@ const makeCanonicalReq = (req: Request) => {
   });
 
   // TODO: make this stage prefixing logic better? (yuck)
-  const pathPrefix = config.stage !== 'prd' ? `/${config.stage}` : '';
+  const pathPrefix = config.stage !== 'prd' && config.stage !== 'test' ? `/${config.stage}` : '';
```

Comment:
I noticed this logic when trying to troubleshoot early on, and I figure it must be designed to handle something different between prod/local and so the logic for prod should apply to staging as well.

Reply:
Have to agree with that previous line TODO, ha. Wdyt:
```suggestion
  const pathPrefix = ['prd', 'test'].includes(config.stage) ? '' : `/${config.stage}`;
```